INSTALL.md

```diff
@@ -2,11 +2,24 @@ HERA M&C Installation
 =====================
 
 Installation of the Python code is just done with a standard
-
 ```
 python setup.py install
 ```
 
+# Begin simplified cm-only user installation
+*If you are using hera_mc to view configuration, just follow the instructions in this short section*
+
+Clone the following two repositories:
+https://github.com/HERA-Team/hera_mc
+https://github.com/HERA-Team/hera_cm_db_updates
+Then install by:
+  1 - within the hera_mc directory type `python setup.py install`
+  2 - in the parent directory of hera_cm_db_updates type `mc_setup_home.py`
+To run hera_mc, you will likely need to install some additional python modules (see Python Prerequisites below.)
+
+To test if it works, try `geo.py -g`
+# End simplified cm-only user installation
```

Comment:
This section of the file is not very readable [when rendered](https://github.com/HERA-Team/hera_mc/blob/054bff553bc9f698bc3086198d6dd8f69ff7e8be/INSTALL.md) since it's not following the [Markdown formatting guidelines](https://guides.github.com/features/mastering-markdown/). Cleaning it up should be pretty straightforward.